frontend/app/src/components/accounts/management/AccountForm.vue

```diff
@@ -31,6 +31,38 @@ const errors = defineModel<ValidationErrors>('errorMessages', { required: true }
 const chain = useRefPropVModel(modelValue, 'chain');
 
 const { isEvm } = useSupportedChains();
+const { t } = useI18n();
+const { apiKey, load: loadApiKeys } = useExternalApiKeys(t);
+const router = useRouter();
+
+const etherscanApiKeyAlert = computed(() => {
+  const selectedChain = get(chain);
+  const currentModelValue = get(modelValue);
+
+  if (
+    selectedChain
+    && get(isEvm(selectedChain))
+    && currentModelValue.mode === 'add'
+    && 'evm' in currentModelValue
+    && currentModelValue.evm
```

Comment:
Hmmm from the discussion we had we decided to show this message if evm is not selected right? other than that it seems fine.

Reply:
so every chain should see this warning even if they dont support etherscan like bitcoin?

Reply:
> so every chain should see this warning even if they dont support etherscan like bitcoin?

No the only thing you need to do is inverse the currentModelValue.evm, so `!currentModelValue.evm`

So it will show for all changes if the evm checkbox is not checked.
So if you are adding an ethereum only account, or an optimism only account and the checkbox for evm is not checked the message should appear but if evm is checked it should not.